meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -498,6 +498,8 @@ PREFERRED_VERSION_optee-os-fio-mfgtool:mx93-nxp-bsp = "3.21.0-imx"
 PREFERRED_VERSION_optee-os-tadevkit:mx93-nxp-bsp = "3.21.0-imx"
 PREFERRED_PROVIDER_virtual/kernel:mx93-nxp-bsp ?= "linux-lmp-fslc-imx"
 PREFERRED_VERSION_u-boot-fio:mx93-nxp-bsp ?= "imx-2023.04"
+# linux-lmp-fslc-imx_6.1 does't have this devicetrees
+KERNEL_DEVICETREE:remove:mx93-nxp-bsp = "freescale/${KERNEL_DEVICETREE_BASENAME}-i2c-spi-slave.dtb freescale/${KERNEL_DEVICETREE_BASENAME}-iw612-otbr.dtb"
```

Comment:
We should be using the updated kernel here (6.6 based).

Reply:
I will drop this from part 3 and return on part 4 after testing